src/chapters/methods/studio.rst

```diff
@@ -0,0 +1,88 @@
+Studio: Methods
+====================
+
+The ``Reverse`` method flips the order of the elements within an array.
```

Comment:
actually, having a note in the intro to the chapter on these built in array methods and how they have seen some methods before would be a great way to ease the reader into the new topic of what methods are. you can link back to the array methods appendix